### PR TITLE
Discover Groups component

### PR DESCRIPTION
**Pull request by ZakkMan**
Build out component described in #324. 

I had to import this thing to make it so the component would work.

### CODE REVIEW

File: client/src/components/DiscoverGroups/DiscoverGroups.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import DiscoverGroups from './DiscoverGroups.js'
+import Adapter from 'enzyme-adapter-react-16';
+configure({adapter: new Adapter()});
+
+// We will describe a block of tests
+describe('DiscoverGroups component', () => {
+	// we will write one individual test
+  it('should render as expected', () => {
+    const component = mount(<DiscoverGroups />)
+		expect(false).toBe(true);
```
Comment (Underwaterr) [inline client/src/components/DiscoverGroups/DiscoverGroups.test.js]: 🤔

File: client/src/components/DiscoverGroups/DiscoverGroups.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.DiscoverGroups {
+
+}
```
Comment (Underwaterr) [inline client/src/components/DiscoverGroups/DiscoverGroups.css]: No need for an empty CSS class, right?

**Comment by hisstori** [comment]
- Related Issue (include '#'): Issue: #324 , there is also another component in that issue that I will finish. It will be on a separate branch.

- Description of changes made: Adjusted the text flow, text size and finished polishing out the "Discover New Groups" component card.

- Is the feature complete/bug resolved/etc..: Feature complete.

- Any known bugs/strange behavior: No known bugs.

- Is there specific feedback you would like on these changes:

- Screenshot(s):

**Comment by hisstori** [comment]
@Underwaterr @ZakkMan

**Pull request merged**